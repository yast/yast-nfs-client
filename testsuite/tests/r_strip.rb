# encoding: utf-8

# Module:
#   NFS client configuration
#
# Summary:
#   Routines testuite
#
# Authors:
#   Martin Vidner <mvidner@suse.cz>
#
# $Id$
module Yast
  class RStripClient < Client
    def main
      # testedfiles: routines.ycp Testsuite.ycp

      Yast.include self, "testsuite.rb"
      Yast.include self, "nfs/routines.rb"

      DUMP("StripExtraSlash")
      TEST(->() { StripExtraSlash("") }, [], nil)
      TEST(->() { StripExtraSlash("/") }, [], nil)
      TEST(->() { StripExtraSlash("/normal/path") }, [], nil)
      TEST(->() { StripExtraSlash("/trailing/slash/") }, [], nil)

      nil
    end
  end
end

Yast::RStripClient.new.main
