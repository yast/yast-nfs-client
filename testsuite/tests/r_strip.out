Dump	StripExtraSlash
Return	
Return	/
Return	/normal/path
Return	/trailing/slash
